### PR TITLE
(feat) Added notifications if you're not on chrome or don't have the ext...

### LINKED ISSUE #258: Chrome Extension: if not installed, tell player to install chromecast extension

### PR DESCRIPTION
**Pull request by rmagee88**
...ension :fire: :droplet: 
fix #258

### CODE REVIEW

File: client/chromecast-app/waiting/waiting.js [hunk 1]
```diff
@@ -13,7 +13,7 @@
       vm.mainContent = {
         waitingForPlayers: {
           text: 'Where are your friends? You\'ve gotta have at least 3!',
-          image: 'http://cdn.meme.am/instances/500x/60894524.jpg'
+          image: 'https://i.imgflip.com/kj7ry.jpg'
```
Comment (ajkitson) [inline client/chromecast-app/waiting/waiting.js:16]: Do we have a less pixelated version of bad luck brian?

**Comment by ajkitson** [comment]
Agreed on :fire: :droplet: , but on reflection I think it'd cleaner to put the check for being on Chrome and having the Cast extension in it's own service, separate from the messaging platform. The controller can just check in with that service directly when it's ready. That keeps questions like 'is the user ready' in the controller and let's the messaging platform just worry about messaging.

File: client/sender-app/start/start.html [hunk 2]
```diff
@@ -1,6 +1,15 @@
 <section class="container-fluid body-content">
   <div class="controls small-center">
-    <img ng-show="!vm.connectionStatus" ng-click="vm.connect()" src="content/ic_cast_grey_24dp.png" alt="chromecast icon">
+    <img ng-show="!vm.connectionStatus" ng-click="vm.hasExtension && vm.onChrome && vm.connect()" src="content/ic_cast_grey_24dp.png" alt="chromecast icon">
+    <div class="modal-content" ng-show="!vm.onChrome" >
+      <p class="start-modal">Ummm, this is awkward...</p>
+      <p class="start-modal">You'll need Google Chrome to play this on a laptop/desktop...  <a href="https://www.google.com/chrome/browser/desktop/index.html" target="_blank">Download Chrome</a> to get it.</p>
+      <p class="start-modal">Once you've downloaded Chrome head over to the <a href="https://chrome.google.com/webstore/detail/google-cast/boadgeojelhgndaghljhdicfkmllpafd?hl=en" target="_blank">Chrome Web Store</a> and download the Google Cast Extension, you'll need that too!</p>
+    </div>
```
Comment (ajkitson) [inline client/sender-app/start/start.html:8]: Do we want to have the message account for the fact that many people will have chrome? Could just change it to "Download Chrome if you don't already have it or fire it up if you do".

Or maybe at the end a message in parens: "(Or maybe you already have Chrome installed. In that case, fire it up. You'll need the Chromecast extension. If you don't have it, we'll give you a link once you're over in Chrome)"

File: client/chromecast-app/waiting/waiting.js (hunk 1 above)
Comment (rmagee88) [inline client/chromecast-app/waiting/waiting.js:16]: Checked, and doesn't seem to be a better option...

**Comment by rmagee88** [comment]
Updated per your comments @ajkitson - think it's simpler now... assumes people are good, and when they click on our icon we'll tell them what needs to happen.  No timeouts, just based on user interaction... works!

**Comment by i-made-that** [comment]
lgtm :pager: :palm_tree: :facepunch:

**Pull request merged**